src/app/icd/server/ICDConfigurationData.h

```diff
@@ -123,6 +125,8 @@ class ICDConfigurationData
 
     static constexpr System::Clock::Seconds32 kMaxIdleModeDuration = System::Clock::Seconds32(18 * kSecondsPerHour);
     static constexpr System::Clock::Seconds32 kMinIdleModeDuration = System::Clock::Seconds32(1);
+    // As defined in the spec, the maximum duration for the StayActiveDuration is 30s (spec 7.5.1)
```

Comment:
Which spec?  7.5.1 in the main Matter spec is "7.5.1. Accessing Fabric"...

If you are looking at spec bits directly on github, the section numbers will be variously wrong, generally.